### ISSUE TITLE
OAK-D-LITE

### ISSUE THREAD

**Issue by Luxonis-Brandon** [luxonis/depthai-hardware#55]
### Start with the `why`:

TLDR: Get DepthAI to the masses. 

The existing DepthAI line has proven useful to many, but so many have lost out on learning about its capability and being able to build with it because it is too expensive for them to justify without knowing for sure if they can pull off what they want with it.  At $199 for the lowest-price DepthAI, that's a big ask for a lot of people.  Particularly people who don't immediately know if it works for their application.

This problem is confounded by the fact that we're not good at marketing or sales.  So we have not done a good job showing people that it is good for their application.  So we can solve the marketing problem to solve this, or we can make a device that is low-cost enough that people can buy it even if they don't yet know if it will work for them.  

So if we make this low-cost variant, and price it low enough, people will buy it not yet knowing if it solves their problem.  

This enables folks to then buy it and use if for things for which we did not even know it would be useful.  If it's low enough cost, it enables all sorts of engineers to take the plunge and then educate _us_ about how the thing can be used.  Folks will find all sorts of new uses for it.

And importantly, it gets it into the hands of folks who have discretionary income for a $99 device, but not for a $199 device.  Which is likely a LOT more people.  

### Move to the `how`:

So how in the world can we make a $99 DepthAI?  Before, there was no way, we couldn't even produce the device for that - mainly because we didn't have the necessary degrees of freedom on camera modules.

But with ArduCam now working closely with us, we can use slightly-lower-spec image sensors that are approximately an order-of magnitude less expensive, which then allows us to hopefully build a complete, 3-camera DepthAI that is less than $99 MSRP.  Which is huge in terms of folks being able to buy it at risk compared to $199 MSRP.

And this might allow us to hit $75 KickStarter Early-Bird for example.  But even at $99, this is allows so many more folks to buy and experiment with it.

And pivotally, many applications, maybe something like 90% of them, can deal with these lower-spec modules without a problem.  

### Move to the `what`:

The OAK-D-LITE, with the following specs:

- OV7251 (640x480) instead of OV9282 (1280x800)
- IMX214 (12MP) instead of IMX378 (12MP), just lower sensitivity and other specs.
- No enclosure, just the PCBA.
- Remove the power jack (less complex power electronics, but requires USB3-capable host, or USB2 host that can provide 900mA - RPi3 and RP4 can, for exmaple).

Simplify and cost-down the design where possible, perhaps not listed above.

**Comment by Luxonis-Brandon** [comment]
Here's some updates on the design:
![image](https://user-images.githubusercontent.com/32992551/115608851-08bef280-a2a4-11eb-8abd-01604813a2ff.png)
![image](https://user-images.githubusercontent.com/32992551/115608835-0361a800-a2a4-11eb-9d7a-72082259aefd.png)
![image](https://user-images.githubusercontent.com/32992551/115608871-0d83a680-a2a4-11eb-8bbd-868418e4eadb.png)

**Comment by Luxonis-Brandon** [comment]
![DM1098LUXD_LITE_R0M0E0 pcb](https://user-images.githubusercontent.com/32992551/119376865-b34a7c80-bc79-11eb-8720-b0d617d8c031.jpg)

**Comment by Luxonis-Brandon** [comment]
![pcba DM1098LUXD_LITE_R0M0E0](https://user-images.githubusercontent.com/32992551/119922922-02203c80-bf2e-11eb-9a57-7cca4223a143.jpg)

**Comment by Luxonis-Brandon** [comment]
Initially working:
![image](https://user-images.githubusercontent.com/32992551/120665680-3b364080-c449-11eb-9e8c-85ced7a4a594.png)
![image](https://user-images.githubusercontent.com/32992551/120665723-42f5e500-c449-11eb-9742-458a2e13dc14.png)

**Comment by Luxonis-Brandon** [comment]
![Uploading IMG_20210603_171345.jpg…]()

**Comment by Luxonis-Brandon** [comment]
![image](https://user-images.githubusercontent.com/32992551/120687237-0c2ac980-c45f-11eb-9307-837a194cb3d0.png)

**Comment by Luxonis-Brandon** [comment]
Implemented in https://github.com/luxonis/depthai-hardware/pull/57
